templates/vpc/content/vpc.tf

```diff
@@ -0,0 +1,9 @@
+#
+# VPC
+#
+
+module "vpc" {
+  source            = "github.com/cds-snc/terraform-modules//vpc?ref=v9.2.5"
+  name              = ${{ values.vpc_name | dump }}
+  billing_tag_value = ${{ values.billing_tag_value| dump }} 
```

Comment:
For the configuration options available I have a few more to request, and I get that this is probably awkward to understand via a comment so more than happy to chat about it tomorrow:

1. High availability?  (true/false)
```hcl
  # true
  availability_zones = 3
  single_nat_gateway = false

  # false
  availability_zones = 1 
  single_nat_gateway = true
```

2. Allow HTTPS requests in (true/false)?  Should users and internet services be able to make requests to something running in the VPC and get a response?
```hcl
  # true
  allow_https_request_in           = true
  allow_https_request_in_response  = true

  # false
  allow_https_request_in           = false
  allow_https_request_in_response  = false
```

3. Allow HTTPS requests out (true/false)?  Is there a service running in the VPC that will be calling out to internet services and expect a response?
```hcl
  # true
  allow_https_request_out          = true
  allow_https_request_out_response = true

  # false
  allow_https_request_out          = false
  allow_https_request_out_response = false
```

Reply:
This has been changed.